src/test/scala/beam/integration/CarSharingSpec.scala

```diff
@@ -105,8 +106,11 @@ class CarSharingSpec extends FlatSpec with Matchers with BeamHelper {
 
     // Only driving allowed
     val population = scenario.getPopulation
+    val nonCarModes = BeamMode.allBeamModes map { mode =>
+      if (mode == BeamMode.CAR) "" else mode.toString.toLowerCase
+    } mkString ","
```

Comment:
this could lead to nonCarModes=",ride_hail,..." => avoid leading/trailing ","

Reply:
there will be no trailing or leading coma in this case , as we are making a list as csv string

I choose this because i thought it would be good to fetch all modes from the Beam Mode definitions rather than harcoding it here in the spec as "ride_hail,bus,train......"
@wrashid